voyageai/embeddings_utils.py

```diff
@@ -74,7 +74,7 @@ async def _aget_embeddings(
     return [d["embedding"] for d in data]
 
 
-def _check_input_type(input_type: str):
+def _check_input_type(input_type: Optional[str]):
```

Comment:
Thanks for the change! This is good, but the functions in this file have been deprecated. Have you checked other files, such as client.py and client_async.py?

Reply:
The type checker does complain about `client_async.py`, but I think it may be related to funkiness with `tenacity` as opposed to the `voyageai` package. I didn't dig deep into it, but I'll take another look tonight and push another PR with some fixes.

Reply:
Great, thank you!